lmnet/.gitignore

```diff
@@ -1,32 +1,18 @@
 # osx
 .DS_Store
 
-
 # python
 *.pyc
 __pycache__/
 
-
 # pyenv
 .python-version
 
-
 # project
 dataset/
 tmp/
 saved/
 
-
-# personal
```

Comment:
Why do you delete these settings ?

Reply:
These are not project specific files. I suppose we should use `gitignore_global` by each developer.

Reply:
OK. In another PR, It is better to fix this, I think. But no problem, thanks 👍